*** ifdh_wrap.cxx.orig	2013-03-28 11:06:05.237183000 -0500
--- ifdh_wrap.cxx	2013-03-28 11:07:38.087872000 -0500
***************
*** 2048,2054 ****
      void *vptr = 0;
      
      /* here we get the method pointer for callbacks */
!     char *doc = (((PyCFunctionObject *)obj) -> m_ml -> ml_doc);
      const char *desc = doc ? strstr(doc, "swig_ptr: ") : 0;
      if (desc) {
        desc = ty ? SWIG_UnpackVoidPtr(desc + 10, &vptr, ty->name) : 0;
--- 2048,2054 ----
      void *vptr = 0;
      
      /* here we get the method pointer for callbacks */
!     const char *doc = (((PyCFunctionObject *)obj) -> m_ml -> ml_doc);
      const char *desc = doc ? strstr(doc, "swig_ptr: ") : 0;
      if (desc) {
        desc = ty ? SWIG_UnpackVoidPtr(desc + 10, &vptr, ty->name) : 0;
***************
*** 3796,3802 ****
  SWIG_AsCharPtrAndSize(PyObject *obj, char** cptr, size_t* psize, int *alloc)
  {
    if (PyString_Check(obj)) {
!     char *cstr; int len;
      PyString_AsStringAndSize(obj, &cstr, &len);
      if (cptr)  {
        if (alloc) {
--- 3796,3802 ----
  SWIG_AsCharPtrAndSize(PyObject *obj, char** cptr, size_t* psize, int *alloc)
  {
    if (PyString_Check(obj)) {
!     char *cstr; Py_ssize_t len;
      PyString_AsStringAndSize(obj, &cstr, &len);
      if (cptr)  {
        if (alloc) {
***************
*** 9431,9441 ****
      swig_type_info **types_initial) {
      size_t i;
      for (i = 0; methods[i].ml_name; ++i) {
!       char *c = methods[i].ml_doc;
        if (c && (c = strstr(c, "swig_ptr: "))) {
          int j;
          swig_const_info *ci = 0;
!         char *name = c + 10;
          for (j = 0; const_table[j].type; ++j) {
            if (strncmp(const_table[j].name, name, 
                strlen(const_table[j].name)) == 0) {
--- 9431,9441 ----
      swig_type_info **types_initial) {
      size_t i;
      for (i = 0; methods[i].ml_name; ++i) {
!       const char *c = methods[i].ml_doc;
        if (c && (c = strstr(c, "swig_ptr: "))) {
          int j;
          swig_const_info *ci = 0;
!         const char *name = c + 10;
          for (j = 0; const_table[j].type; ++j) {
            if (strncmp(const_table[j].name, name, 
                strlen(const_table[j].name)) == 0) {
